### PR TITLE
WIP -  dast test

### PR DESCRIPTION
**Pull request by obochan-rh**

### CODE REVIEW

File: ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__telco-operators-dast-4.16.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      ADDITIONAL_WORKERS: "0"
+      ADDITIONAL_WORKERS_DAY2: "false"
+      AGENT_PLATFORM_TYPE: none
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      DISCONNECTED: "false"
+      OPERATORS: |
+        [
+          {"name": "local-storage-operator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-local-storage", "target_namespaces": "openshift-local-storage", "operator_group": "local-operator-group"},
+          {"name": "advanced-cluster-management", "source": "redhat-operators", "channel": "release-2.11", "install_namespace": "open-cluster-management", "target_namespaces": "open-cluster-management", "operator_group": "acm-operator-group"},
+          {"name": "topology-aware-lifecycle-manager", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-talm", "target_namespaces": "", "operator_group": "topology-aware-lifecycle-manager"},
```
Comment (shaior) [inline ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__telco-operators-dast-4.16.yaml:45]: Duplicate, use the last instance of this operator on line 50.

File: ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__telco-operators-dast-4.16.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          {"name": "topology-aware-lifecycle-manager", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-marketplace","target_namespace": "openshift-operator-lifecycle-manager", "operator_group": "redhat-operators"},
+        ]
+      RESERVE_BOOTSTRAP: "false"
+      architecture: amd64
+      ipv4_enabled: "true"
+      ipv6_enabled: "true"
+      masters: "1"
+      workers: "0"
+    test:
+    - ref: telcov10n-functional-dast-tests
```
Comment (shaior) [inline ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__telco-operators-dast-4.16.yaml:59]: Please add install operators refs here.
ref: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/distributed-tracing-qe/openshift-distributed-tracing-qe-main__ocp-4.14-acceptance.yaml#L48

**Comment by shaior** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by shaior** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by hector-vido** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Comment by obochan-rh** [comment]
/pj-rehearse periodic-ci-openshift-kni-eco-ci-cd-main-telco-operators-dast-4.16-telco-dast-ci

**Pull request closed without merging**